// [python quickstart](https://grpc.io/docs/quickstart/python.html#run-a-grpc-application)
// python -m grpc_tools.protoc --python_out=. --grpc_python_out=. -I. xxxxxxx.proto


syntax = "proto3";

service SalientObjectSegm {
    rpc infer_one_img(SOSOneImgRequest) returns (SOSOneImgReply) {}
}

message SOSOneImgRequest {
    bytes img_bgr_bytes = 1;
}

message SOSOneImgReply {
    bytes result_bytes = 1;
}


